gno.land/pkg/integration/testing_integration.go

```diff
@@ -211,10 +230,73 @@ func setupGnolandTestScript(t *testing.T, txtarDir string) testscript.Params {
 
 				tsValidateError(ts, "gnokey", neg, err)
 			},
+			"sleep": func(ts *testscript.TestScript, neg bool, args []string) {
+				d := time.Second
+				if len(args) > 0 {
+					var err error
+					if d, err = time.ParseDuration(args[0]); err != nil {
+						ts.Fatalf("unable to parse duration %q: %s", args[1], err)
+					}
+				}
+
+				time.Sleep(d)
+			},
+			// adduser commands must be executed before starting the node; it errors out otherwise.
+			"adduser": func(ts *testscript.TestScript, neg bool, args []string) {
```

Comment:
this is a good one, but in addition to adding it, I'd prefer having a bunch of well known accounts already created too.

Reply:
I personally like it because setting this at the beginning of the test provides more context about the different actors involved in the test.

Reply:
I think if we want "well known" accounts created then maybe they should be added to the genesis file.  Or perhaps these tests should have their own genesis file; not the main one that is currently checked in.

Reply:
We already have a hardcoded test1 in txtar. My goal is to proceed with test2, test3, and test4. We can include the same in the genesis file later. However, I was specifically talking about txtar, where we don't need to consider the genesis but instead want to simulate multiuser transactions.

Reply:
Yeah that's fine; I'll add them.

Reply:
I've added them.